docusaurus/docs/dev-docs/api/rest.md

```diff
@@ -26,21 +26,21 @@ For each Content-Type, the following endpoints are automatically generated:
 
 | Method   | URL                             | Description                           |
 | -------- | ------------------------------- | ------------------------------------- |
-| `GET`    | `/api/:pluralApiId`             | [Get a list of entries](#get-entries) |
-| `POST`   | `/api/:pluralApiId`             | [Create an entry](#create-an-entry)   |
-| `GET`    | `/api/:pluralApiId/:documentId` | [Get an entry](#get-an-entry)         |
-| `PUT`    | `/api/:pluralApiId/:documentId` | [Update an entry](#update-an-entry)   |
-| `DELETE` | `/api/:pluralApiId/:documentId` | [Delete an entry](#delete-an-entry)   |
+| `GET`    | /api/:pluralApiId             | [Get a list of entries](#get-entries) |
```

Comment:
Is it intended to remove the backquotes in the table? 🤔

Reply:
I've follow the Design in figma, but I can revert easily if you want, @pwizla

Reply:
I think Maeva simply forgot to re-add the inline code style 😅  :D 
Please see my comment [in Jira](https://strapi-inc.atlassian.net/browse/DOCX-261?focusedCommentId=15687) 🙂